dune-project

```diff
@@ -35,8 +35,8 @@
  (name atd)
  (depends
   (ocaml (>= 4.08))
-  (dune (>= 2.0))
-  menhir
+  (dune (>= 2.8))
```

Comment:
you can remove all the `(dune (>= 2.8))` from dune-project because dune automatically adds this dependency

Reply:
ok, great. The fix is here: https://github.com/ahrefs/atd/pull/260